installAndGrantPermission.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+if ! command -v adb &> /dev/null
+then
+    echo "ADB needs to be in your PATH. The script will finish now."
+    echo "Head over https://developer.android.com/studio/command-line/adb for more info about ADB."
+    echo "Here's a good step-by-step guide on how to setup ADB in your PATH: https://wiki.lineageos.org/adb_fastboot_guide.html"
+    exit
+fi
+
+adb install -t -r -g proxy-toggle.apk
```

Comment:
Maybe the script itself can build the latest app, grant the permission and install it on the device.

Reply:
Oh, that's a good idea - will try to think about this when running from Android Studio.

However, this script is more intended for when releasing the app, we will bundle the apk and this script into a .zip file... 
So let's say a tester that doesn't have Android Studio or gradle knowledge, they will only have to download the zip file and execute the script and viola! the app is installed with permissions 😄

Reply:
Oh yes that makes more sense actually